frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -52,6 +52,8 @@ interface State {
    * The value specified by the user via the UI.
    */
   value: number[]
+
+  overMaxSelections: boolean | undefined
```

Comment:
Instead of storing this in the state and having to calculate-and-set it all over the place, could we make it a method or computed property that checks `this.state.value.length` and `this.maxSelections`?

That way, we wouldn't have to figure out what to set it to everywhere, we could just set the value and this would be calculated correctly for free.

Reply:
Extremely good point! Thanks!